theme_anelusia/views/images_library.xml

```diff
@@ -242,5 +242,25 @@
     <field name="name">website.s_accordion_image_default_image</field>
     <field name="url">/theme_anelusia/static/src/img/snippets/s_picture.jpg</field>
 </record>
+<record id="s_wavy_grid_default_image_1" model="theme.ir.attachment">
```

Comment:
Some images are duplicated, idk if it's intended ? 
![Screenshot 2024-08-26 at 11 48 04](https://github.com/user-attachments/assets/8556ebc8-a7d7-4816-a6db-bce13cbf1f02)

Reply:
Good catch, thanks ! I've fixed the issue 👍